### PR TITLE
features(common):User Interface for Viewing Volume Items[SEM-15]

### PR DESCRIPTION
**Pull request by calebrian**
In this interface, articles from a selected volume are displayed.

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,7 +1,100 @@
...
....28655 38.3584 6.09447 38.0831C3.18523 33.9097 1.92469 29.2867 2.23949 24.2302C2.83441 14.6917 10.1015 6.49649 19.5749 4.77685C25.3467 3.73017 30.773 4.76621 35.6164 8.09643C41.0867 11.8589 44.3094 17.087 44.859 23.7288C45.582 32.4614 42.0431 39.2043 34.7347 43.9669C32.0269 45.7317 28.9776 46.6521 25.7469 46.9859C22.3027 47.341 18.992 46.8396 15.7613 45.6453C15.4771 45.5402 15.069 45.6227 14.7662 45.7384C11.2487 47.0883 7.7405 48.4621 4.22834 49.828C4.04826 49.8985 3.86152 49.9557 3.56272 50.0594ZM4.8406 48.6244C5.16741 48.5153 5.3795 48.4542 5.58492 48.3744C8.59954 47.2 11.6168 46.0363 14.6208 44.838C15.173 44.6172 15.6332 44.6146 16.1948 44.8353C19.7203 46.2198 23.3912 46.6587 27.0941 45.9246C34.8027 44.3951 40.1477 39.953 42.9355 32.5824C44.2067 29.2216 44.4655 25.7291 43.8266 22.2073C42.9982 17.6482 40.8973 13.7315 37.4198 10.6353C32.2643 6.0443 26.2271 4.42042 19.4615 5.70516C15.2957 6.49649 11.7115 8.48212 8.80363 11.5769C3.94822 16.7425 2.20081 22.8497 3.46801 29.7988C4.01491 32.8032 5.3008 35.507 7.10557 37.9714C7.24563 38.1629 7.37635 38.4648 7.32833 38.6736C7.04154 39.9424 6.69606 41.1979 6.37992 42.46C5.87437 44.4696 5.3755 46.4792 4.8406 48.6231V48.6244Z" fill="white"/>
```
Comment (Ditmar) [inline src/app/page.tsx:73]: SVGs and Accessibility Consider extracting SVGs into separate components or using an image asset management library.

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,7 +1,100 @@
...
+                                  <path d="M6.75642 17.0086C6.75642 16.2558 6.7616 15.503 6.75297 14.7511C6.75125 14.5744 6.8004 14.5248 6.97631 14.5283C7.57044 14.5396 8.16543 14.5266 8.76043 14.5353C9.91333 14.5527 10.5359 15.067 10.78 16.2114C10.8981 16.7675 10.8946 17.3288 10.7653 17.8832C10.5368 18.864 9.90212 19.4088 8.90357 19.4506C8.25339 19.4775 7.60148 19.454 6.94958 19.4654C6.77108 19.468 6.75125 19.3836 6.75384 19.2391C6.75987 18.8675 6.75556 18.4959 6.75556 18.1243C6.75556 17.7527 6.75556 17.3811 6.75556 17.0094L6.75642 17.0086ZM7.75842 16.9799C7.75842 17.4568 7.76101 17.9328 7.7567 18.4097C7.75584 18.5429 7.7774 18.6238 7.94037 18.6168C8.24218 18.6038 8.54571 18.6221 8.84666 18.6047C9.333 18.5751 9.6167 18.3436 9.73225 17.8675C9.86504 17.3236 9.86763 16.7693 9.75208 16.2236C9.63998 15.6936 9.31489 15.4299 8.77853 15.3907C8.54312 15.3742 8.30599 15.3847 8.07144 15.3603C7.83258 15.3359 7.73687 15.3994 7.75066 15.6658C7.77481 16.1026 7.75756 16.5421 7.7567 16.9807L7.75842 16.9799Z" fill="#178435"/>
```
Comment (Ditmar) [inline src/app/page.tsx:41]: SVGs and Accessibility Consider extracting SVGs into separate components or using an image asset management library.

**Comment by olmedo90** [comment]
image

It is according to the figma. continue with the same interest. 👍

File: src/components/Articles/index.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    };
+
+    fetchDataAndSetState();
+  }, []);
+
+    return (
+        <div className="flex min-h-screen items-top justify-between p-24">
+            <div className=" w-3/4 mr-16">
+                {articles.slice(0, 10).map((article: Article) => (
+                    <div key={articles.id} className="">
```
Comment (Ditmar) [inline src/components/Articles/index.tsx:28]: Ok, create another component for every item of the component in a separate file.

File: src/components/Articles/scss/mystile.scss [hunk 4]
```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```
Comment (Ditmar) [inline src/components/Articles/scss/mystile.scss:1]: Please read this article of the next documentation to use to optimize fonts for next 13 https://nextjs.org/docs/pages/building-your-application/optimizing/fonts

**Review by Ditmar — CHANGES_REQUESTED** [review]
Good improvements, only two comments please check it

File: src/app/async/api.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+export const fetchData = async (): Promise<Article[]> => {
+    try {
+        const response = await fetch('https://jsonplaceholder.typicode.com/comments');
```
Comment (Ditmar) [inline src/app/async/api.ts:3]: ok is good but please don't use directly the URL you need to pass that as a parameter 
```
export const fetchData = async (URL: string): Promise<Article[]> => {
    try {
        const response = await fetch(URL);
        ...
```
and you can use it as follows
```
fetchData (`process.env.BASE_API_URL/${service}`)
```
this is only an example you need to made changes based in your requirements

**Review by Ditmar — CHANGES_REQUESTED** [review]

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**